src/Amara/Varcon/TranslationProvider.php

```diff
@@ -43,10 +43,6 @@ public function getTranslations($from, $to, $threshold = 80)
     {
         $fileHandle = fopen($this->filePath, 'r');
 
-        if (!$fileHandle) {
-            throw new \RuntimeException(sprintf('Unable to open file: %s', $filename));
```

Comment:
Though in the case of failure, fopen is already throwing an exception before we can reach this line to throw ours. ¯\_(ツ)_/¯

We could add a file_exists before fopen, and throw one there I guess - do you think this would make sense?